Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandCloakEnableDisable.java

```diff
@@ -51,7 +51,7 @@ public final void run(final Map<String, String> parameters)
 		{
 			character.setProperty("cloaked", false);
 			
-			addCallbackData("html", "<a onclick='toggleCloaked(event)'  title='Clicking here will hide your equipment and stats from other players.'><img src='images/ui/cloakedDisabled.png' border=0/></a>");
+			addCallbackData("html", "<a onclick='toggleCloaked(event)'  title='Clicking here will show your equipment and stats to other players.'><img src='images/ui/cloakedDisabled.png' border=0/></a>");
```

Comment:
I'd like to take this opportunity to use the new "updateHtml()" methods that I added this morning. Give that a shot instead of the generic callbackdata method.